lib/push.js

```diff
@@ -129,10 +129,26 @@ module.exports = function (options) {
 
       var files = wrench.readdirSyncRecursive('.');
       var excludedPaths = ['.build', '.git', '.whdist', '.push.zip'];
-      if (!custom)
-      {
+      if (!custom) {
         excludedPaths = excludedPaths.concat('package.json', 'node_modules', 'Gruntfile.js', 'libs', 'tasks', 'options');
       }
+      if (fs.existsSync('webhookconfig.json')) {
+        try {
+          var contents = JSON.parse(fs.readFileSync('webhookconfig.json', 'utf8'));
```

Comment:
should probably declare the contents variable outside of the try block, even if it technically wont crash because the declaration gets hoisted